src/Blocks/BlockChest.h

```diff
@@ -33,21 +33,27 @@ class cBlockChestHandler :
 		BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta
 	) override
 	{
-		a_BlockType = m_BlockType;
-
```

Comment:
Why remove this? This way `a_BlockType` can be undefined upon leaving the function.

Reply:
Because a_BlockType will be reset by super::GetBlockTypeMeta, and it would be confusing to have it set here as well when the value will be overridden later.

Reply:
But there's no call to `super:GetBlockTypeMeta`...

Reply:
Look down a bit... If we return false before setting a_BlockType then there should be no issue, right? The placement has failed so the caller has no right to see what block has been placed.

Reply:
Ook, I was looking at the wrong source.